src/crimson/net/SocketConnection.h

```diff
@@ -135,40 +57,20 @@ class SocketConnection : public Connection {
   ///          false otherwise.
   bool update_rx_seq(seq_num_t seq);
 
-  seastar::future<> read_message();
-
-  std::unique_ptr<AuthSessionHandler> session_security;
-
   // messages to be resent after connection gets reset
   std::queue<MessageRef> out_q;
   // messages sent, but not yet acked by peer
   std::queue<MessageRef> sent;
-  static void discard_up_to(std::queue<MessageRef>*, seq_num_t);
-
-  struct Keepalive {
-    struct {
-      const char tag = CEPH_MSGR_TAG_KEEPALIVE2;
-      ceph_timespec stamp;
-    } __attribute__((packed)) req;
-    struct {
-      const char tag = CEPH_MSGR_TAG_KEEPALIVE2_ACK;
-      ceph_timespec stamp;
-    } __attribute__((packed)) ack;
-    ceph_timespec ack_stamp;
-  } k;
 
-  seastar::future<> fault();
-
-  void execute_open();
-
-  seastar::future<> do_send(MessageRef msg);
-  seastar::future<> do_keepalive();
-  seastar::future<> do_close();
+  // which of the peer_addrs we're connecting to (as client)
+  // or should reconnect to (as peer)
```

Comment:
what does "peer" stand for in this context?

Reply:
This is the same comment from `src/async/AsyncConnection.h`, I think this means non-client. Like stateful_server, lossless_peer, or lossless_peer_reuse.